ts/store/reducers/content.ts

```diff
@@ -133,6 +138,21 @@ export const servicesInScopeSelector = (
     )
   );
 
+export const idpTextDataSelector = (
```

Comment:
```suggestion
export const idpTextDataSelector = (
  state: GlobalState,
  id: IdentityProviderId,
  locale: Locales
) =>
  pot.getOrElse(
    pot.map(state.content.idpTextData, data => {
      return data[locale][id];
    }),
    undefined
  );
```
Pick the idp data from the current language